tests/ui/asm/naked-with-invalid-repr-attr.stderr

```diff
@@ -0,0 +1,93 @@
+error[E0517]: attribute should be applied to a struct, enum, or union
+  --> $DIR/naked-with-invalid-repr-attr.rs:7:8
+   |
+LL |   #[repr(C)]
+   |          ^
+...
+LL | / extern "C" fn example1() {
+LL | |
+LL | |     unsafe { asm!("", options(noreturn)) }
+LL | | }
+   | |_- not a struct, enum, or union
+   |
+   = note: `#[repr(align(...))]` is valid on functions
```

Comment:
Why are we noting this? It's extraneous -- I put `repr(C)` on an extern block; that has nothing to do with `repr(align)` *or* functions.

Reply:
`extern "C" fn example1() {` isn't an extern block?

Reply:
this isn't an extern block, it's a function block. It wouldn't trigger if it wasn't a function. My reasoning was that the only valid repr on functions is align, so if you put c there you might like to know that align *is* a valid repr, but the one you used is not.

Reply:
> Er, misread that as an extern block.

Reply:
This is a very silly language.